api/spec/models/concerns/spree/webhooks/has_webhooks_spec.rb

```diff
@@ -113,4 +113,25 @@
       end
     end
   end
+
+  describe '.default_webhook_events' do
+    it 'return the default events' do
+      expect(Spree::Order.default_webhook_events).to eq(%w[order.create order.delete order.update])
+    end
+  end
+
+  describe '.supported_webhook_events' do
+    context 'when there are custom supported events' do
+      it 'returns the default and custom events' do
+        all_webhook_events = Spree::Order.default_webhook_events + Spree::Order.custom_webhook_events
```

Comment:
Could you do a fixed check here on the values of `Spree::Order.supported_webhook_events` as you did [here](https://github.com/spree/spree/pull/11495/files#diff-ad2b136c2bb246bb18ee2a64fe1a91c39c2d71b5b12d2eb1e957f1090bcc912cR119)? It might happen that this `Spree::Order.default_webhook_events + Spree::Order.custom_webhook_events` returns an empty array, or any other thing and then you compare it with this `Spree::Order.supported_webhook_events` which might have the proper logic but not returning what you really expect @aleksandarpetrushev.